kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/SolverServicesClientImpl.java

```diff
@@ -100,13 +104,15 @@ public SolverServicesClientImpl(KieServicesConfiguration config, ClassLoader cla
     public ServiceResponse<SolverInstance> updateSolverState(String containerId, String solverId, SolverInstance instance) {
         checkMandatoryParameter( "ContainerID", containerId );
         checkMandatoryParameter( "SolverId", solverId );
+        checkMandatoryParameter( "instance", instance );
```

Comment:
I think that SolverInstance don't need to be mandatory when terminating solver.

Maybe best solution would be to separate starting and terminating of solvers into separate client methods with own mandatory parameter validations.